example/Gemfile

```
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
+ruby ">= 2.6.10"
+
+# Exclude problematic versions of cocoapods and activesupport that causes build failures.
+gem 'cocoapods', '>= 1.13', '!= 1.15.0', '!= 1.15.1'
+gem 'activesupport', '>= 6.1.7.5', '!= 7.1.0'
```

RuboCop (1.69.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `activesupport` should appear before `cocoapods`.

(Bundler/OrderedGems)